lib/twitter_service.rb

```diff
@@ -0,0 +1,20 @@
+require 'twitter'
+
+class TwitterService
+  @@client = Twitter::REST::Client
```

Comment:
Below it says, "No Hound Violations". I think I successfully turned this one off. We need class variables to follow the testing pattern.